configuration/src/wasm/types.rs

```diff
@@ -183,11 +183,6 @@ export interface NomadGasConfig {
   bridge: BridgeGasConfig;
 }
 
-export interface S3Config {
-  bucket: string;
-  region: string;
-}
```

Comment:
Forgot to run build in prev PR?

Reply:
is your branch based on main? the s3 config should definitely be exported to TS now. this deletion is confusing

Reply:
yeah rebased and its gone